paddle/fluid/framework/ir/sync_batch_norm_pass.cc

```diff
@@ -26,7 +26,7 @@ class SyncBatchNormPass : public Pass {
   void ApplyImpl(ir::Graph *graph) const override {
     VLOG(3) << "Use synchronous batch norm";
     for (const Node *n : graph->Nodes()) {
-      if (n->IsOp()) {
+      if (n->IsOp() && n->Op()) {
```

Comment:
Is there a common method to fix this? In fact, all passes should add this check to avoid crash.

Reply:
We should throw an exception in the `Node.Op()` if this Node doesn't have OpDesc. And so, the developer will know the error info.